.snippets/code/learn/dapps-list/dapp-directory/urls.json

```diff
@@ -0,0 +1,5 @@
+"urls": {
+    "website": "https://moonbeam.network/",
+    "try": "https://moonbeam.network/",
```

Comment:
```suggestion
    "try": "https://docs.moonbeam.network/",
```

Reply:
Just so that in the example it shows that "website" and "try" are different, as one is normally the dApp URL